src/main/java/seedu/linkedout/logic/commands/SearchCommand.java

```diff
@@ -22,8 +23,9 @@
     public static final String MESSAGE_USAGE = COMMAND_WORD + ": Search an overview of a specific applicant \n"
             + "Parameters: must contain at least one prefix "
             + "[" + PREFIX_NAME + "NAME] "
-            + "[" + PREFIX_JOB + "JOB] \n"
-            + "Example: " + COMMAND_WORD + " " + PREFIX_NAME + " Steve";
+            + "[" + PREFIX_JOB + "JOB] "
+            + "[" + PREFIX_ROUND + "ROUND] \n"
+            + "Example: " + COMMAND_WORD + " " + PREFIX_NAME + "Steve";
```

Comment:
Might need some of your opinion. Do you think it's better to include all or just some of the prefix (to show AND condition works too)?
```suggestion
            + "Example: " + COMMAND_WORD + " " + PREFIX_NAME + "Steve " + PREFIX_JOB + "Software Engineer " + PREFIX_ROUND + "Intern";
```

Reply:
Hmm I haven't updated this but I think it would probably require quite a complex statement to explain the function... Will try to think of a succinct statement as a description. Thanks for the heads up though!